site/content/usage/experimental/01-gitops.md

```diff
@@ -92,14 +92,54 @@ memcached-958f745c-qdfgz   1/1     Running   0          29m
 ```
 
 
-### Adding a workload
+#### Adding a workload
 
 To deploy a new workload on the cluster using GitOps just add a kubernetes manifest to the repository. After a few 
 minutes you should see the resources appearing in the cluster.
 
-### Further reading
+#### Further reading
 
 To learn more about GitOps and Flux, check the [Flux documentation][flux]
  
 
+### Installing components from a quickstart
+
+`eksctl` provides an application development quickstart profile which can install the following components in your
+cluster: 
+  - Metrics Server
+  - Cortex/dynamo
```

Comment:
We haven't added Cortex with DynamoDB support yet. We should replace this with Prometheus.